src/snippets/typescript.json

```diff
@@ -29,21 +30,22 @@
       "      preview_image: {",
       "        title: intl.formatMessage(messages.preview_image),",
       "        widget: 'object_browser',",
-      "        mode: 'image'",
+      "        mode: 'image',",
       "      },",
       "    },",
       "    required: [],",
       "  };",
       "};",
-      "export default Schema;"
+      "export default Schema;",
+      ""
     ],
     "description": "Volto Block Base Schema"
   },
   "Volto Block Config": {
     "prefix": ["voltoBlockConfig"],
     "body": [
-      "config.blocks.blocksConfig.block06 = {",
-      "  id: 'block06',",
+      "config.blocks.blocksConfig.${1:block06} = {",
```

Comment:
@ionlizarazu do this ${1: dance also in the javascriptreact snippets, or add another PR doing so.

Reply:
I've planned to add this kind of improvements in another PR to separate features